thirdparty/nginx/nginx-1.7.1-zimbra/src/mail/ngx_mail_throttle_module.c

```diff
@@ -350,6 +365,104 @@ ngx_mail_throttle_set_ttl_text (ngx_msec_t ttl, ngx_str_t * input, ngx_str_t * t
    return NGX_CONF_OK;
 }
 
+static char *
+ngx_mail_throttle_whitelist_ips (ngx_conf_t *cf, ngx_command_t *cmd, void *conf) {
```

Comment:
Honestly I just used the same formal parameter list as the pre-existing functions.  This is the same as the `ngx_conf_set_str_slot` function, also used in the config block, and as defined by nginx/core:

    char *ngx_conf_set_str_slot(ngx_conf_t *cf, ngx_command_t *cmd, void *conf);